node/malus/README.md

```diff
@@ -1,3 +1,59 @@
 # malus
 
-Create nemesis nodes with alternate, at best fault, at worst intentionally destructive behavior traits.
\ No newline at end of file
+Create nemesis nodes with alternate, at best faulty, at worst intentionally destructive behavior traits.
+
+The first argument determines the behavior strain. The currently supported are:
+
+* `suggest-garbage-candidate`
+* `back-garbage-candidate`
+* `dispute-ancestor`
+
+## Integration test cases
+
+To define integration tests create file
+in the toml format as used with [zombienet][zombienet]
+under `./integrationtests` describing the network to spawn and
+also the `feature` file (with `.feature` extension ) using the format
+defined in the [DSL doc](https://github.com/paritytech/zombienet/blob/main/docs/testing-dsl-definition.md).
+
+## Usage
+
+> Assumes you already gained permissiones, ping in element @javier:matrix.parity.io to get access.
+> and you have cloned the [zombienet][zombienet] repo.
+
+To launch a test case in the development cluster use (e.g. for the  ./node/malus/integrationtests/0001-dispute-valid-block.toml):
+
+```sh
+# declare the containers pulled in by zombie-net test definitions
+export MALUS_IMAGE=docker.io/paritypr/malus:4131-ccd09bbf
+export ZOMBIENET_INTEGRATION_TEST_IMAGE=docker.io/paritypr/synth-wave:4131-0.9.12-ccd09bbf-29a1ac18
+export COL_IMAGE=docker.io/paritypr/colander:4131-ccd09bbf
```

Comment:
is this going to renamed here or in https://github.com/paritytech/zombie-net/?

Reply:
This should be renamed in `zombienet` and use this names. I will change in that repo to use the same naming.
Thanks!